src/Entity/Config.php

```diff
@@ -14,7 +14,7 @@
  * @ORM\Table(
  *     name="`config`",
  *     indexes={
- *         @ORM\Index(name="config_feed_token", columns={"feed_token"}, options={"lengths"={255}}),
```

Comment:
in most cases except one the removal of the lengths is safe because it match with the column length